library/kani/src/models/mod.rs

```diff
@@ -184,20 +184,6 @@ mod test {
         check_bitmask::<_, [u8; 9], i128, 70>(CustomMask([-1i128; 70]));
     }
 
-    /// Compare the value returned by our model and the portable simd representation.
-    fn check_portable_bitmask<T, E, const LANES: usize>(mask: T)
-    where
-        T: ToBitMask + Clone,
```

Comment:
Actually, I believe you only need to remove lines 190-191, and replace `T::BitMask` by `u64` on line 196.

Reply:
Thanks, done.